de.dashup.application/src/main/webapp/components/dashup-button.js

```diff
@@ -0,0 +1,77 @@
+import {LitElement, html, css} from "https://unpkg.com/lit-element@2.1.0/lit-element.js?module";
```

Comment:
Do you need to import this module in every component, since DashupComponent inherits from LitElement?

Reply:
Removed the unnecessary imports